content/articles/How-to-start-up-with-Socket-Programming-with-Java/index.md

```diff
@@ -202,21 +203,21 @@ inputStreamReader = new InputStreamReader(socket.getInputStream());
 outputStreamWriter = new OutputStreamWriter(socket.getOutputStream());
 ``` 
 
-The Scanner class is used to get input from the user from the console/terminal. System. in is a keyboard input connected input stream system.
+The Scanner class is used to get input from the user from the console/terminal. System.in is a keyboard input connected input stream system.
 
 ```java
 Scanner scanner = new Scanner(System.in);
 ```
 
 ### Methods used in Client Socket
```

Comment:
Change:

>### Methods used in Client Socket

To:

>#### Methods used in Client Socket

Fixing heading structure since this is under: **Programming Client-Side**